src/main/java/org/apache/commons/validator/routines/RegexValidator.java

```diff
@@ -131,6 +131,15 @@ public RegexValidator(String[] regexs, boolean caseSensitive) {
         }
     }
 
+    /**
+     * Getter method for <code>patterns</code>.
+     *
+     * @return the array of <code>patterns</code> used to validate values.
+     */
+    public Pattern[] getPatterns() {
+        return patterns;
```

Comment:
The method must return a copy of the the patterns otherwise the caller can mutate the array.